testing/samurai/APKBUILD

```diff
@@ -0,0 +1,27 @@
+# Contributor: TBK <alpine@jjtc.eu>
+# Maintainer: TBK <alpine@jjtc.eu>
+pkgname=samurai
+pkgver=0.3
+pkgrel=0
+pkgdesc="Ninja-compatible build tool written in C99 with focus on simplicity, speed and portability."
+url="https://github.com/michaelforney/samurai"
+arch="all"
+license="GPL-2.0"
+options="!check" # No test suite
+provides="ninja"
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::https://github.com/michaelforney/samurai/archive/$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	make
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" PREFIX=/usr install
+	ln -s $pkgdir/usr/bin/samu $pkgdir/usr/bin/ninja
```

Comment:
Does this work? It looks like this would create a link with an absolute target of `$pkgdir/usr/bin/samu`, when either `/usr/bin/samu` (absolute link without the pkgdir prefix) or a relative link to `ninja` would be more appropriate.